### PR TITLE
Implement Defence Feedback

### PR DESCRIPTION
**Pull request by abdulsemiu-atanda**
### What does this PR do?
Implements feedback from defence

### Description of task completed

- Test suite consistent name
- no-confusing-arrow fix in source file
- Case insensitivity fix in hasVowels method
- Documentation fix in source file
- Readme.md update

### How to manually test this?

Check files changed

### CODE REVIEW

File: src/stringClass.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const stringClass = {
 
 /**
    * This method takes a string and returns its first letter in uppercase
-   * @param {String} this
+   * It extends the String.prototype to achieve this
```
Comment (q-ode) [inline src/stringClass.js]: This is looking redundant.

Can we leave it out an have a meaningful comment when we're actually assigning the methods?

File: .eslintrc [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
-    "no-confusing-arrow": 0,
```
Comment (q-ode) [inline .eslintrc:19]: I believe you can include the new guide without removing `no-confusing-arrow`. This is still a valid check.

File: src/stringClass.js [hunk 3]
```diff
@@ -1,75 +1,75 @@
...
   },
 
-/**
-   * This method takes a string and returns it in uppercase
-   * @param {String} this
-   * @return {String} this
-   */
+  /**
+     * This method takes a string and returns it in uppercase
+     * It extends the String.prototype to achieve this
```
Comment (q-ode) [inline src/stringClass.js]: `It extends the String.prototype to achieve this` this looks redundant.
Let's take it out.

**Review by q-ode — CHANGES_REQUESTED** [review]
Great job making each change in it's own commit.

File: .eslintrc (hunk 2 above)
Comment (abdulsemiu-atanda) [reply .eslintrc:19]: @q-ode I have updated it and set it as a warning

**Pull request merged**